packages/playwright/src/matchers/expect.ts

```diff
@@ -346,6 +348,7 @@ class ExpectMetaInfoProxyHandler implements ProxyHandler<any> {
       };
 
       try {
+        matcher._pwStepId = step.stepId;
```

Comment:
This looks like a total hack! Perhaps we should follow `setCurrentExpectConfigureTimeout` instead?